lib/test/drm/playready/toBigEndianKeyId.test.ts

```diff
@@ -0,0 +1,42 @@
+import { toBigEndianKeyId } from '@svta/common-media-library/drm';
+import { arrayBufferToUuid, uuidToArrayBuffer } from '@svta/common-media-library/utils';
+import { equal } from 'assert/strict';
+import { describe, it } from 'node:test';
+
+describe('toBigEndianKeyId', () => {
+	it('should convert little-endian key ID to big-endian format', () => {
+		//#region example
+		const littleEndianKeyId = 'a0564af7-60c2-d94b-9610-c7ae70d5d970';
+		const expectedBigEndianKeyId = 'f74a56a0-c260-4bd9-9610-c7ae70d5d970';
+
+		const inputBuffer = uuidToArrayBuffer(littleEndianKeyId);
+		const result = toBigEndianKeyId(inputBuffer);
+		const resultHex = arrayBufferToUuid(result);
+
+		equal(resultHex, expectedBigEndianKeyId);
+		//#endregion example
+	});
+
+	it('should return the same buffer if not 16 bytes', () => {
+		const shortBuffer = new ArrayBuffer(8);
+		const result = toBigEndianKeyId(shortBuffer);
+
+		equal(result, shortBuffer);
+	});
+
+	it('should return the same buffer if longer than 16 bytes', () => {
+		const longBuffer = new ArrayBuffer(32);
+		const result = toBigEndianKeyId(longBuffer);
+
+		equal(result, longBuffer);
+	});
+
+	it('should only swap the first 8 bytes (GUID portion)', () => {
+		const input = uuidToArrayBuffer('00000000-0000-0000-1234-567890abcdef');
+		const result = toBigEndianKeyId(input);
+		const resultHex = arrayBufferToUuid(result);
+
+		// First 8 bytes should be swapped, last 8 bytes should remain unchanged
+		equal(resultHex, '00000000-0000-0000-1234-567890abcdef');
```

Comment:
Sorry, a bit confused, but do we expect anything to change in this test case? Hex string is still the same

Reply:
This was a suggested test, and its use case is already covered by the other tests. Removing it.